### PR TITLE
chore: add setup

### PR DESCRIPTION
**Pull request by co-sic**
This is a public repository and will be published via npm.
Semantic-release will be added in the next PR.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        with:
+          node-version: '14'
+      - name: Cache node modules
+        uses: actions/cache@v2
+        env:
+          cache-name: cache-node-modules
+        with:
+          path: '**/node_modules'
+          key: ${{ runner.os }}-${{ matrix.node-version }}-modules-${{ hashFiles('**/yarn.lock') }}
+      - name: Install dependencies
```
Comment (pommelinho) [inline .github/workflows/ci.yml:35]: i think you can dockerized this.
so make a dockerfile where the se steps are executed.
what do you think @m1pl

Comment (m1pl) [reply .github/workflows/ci.yml:35]: Yes, he already knows. Waiting for me to write an action for it. Dockerfile is prepared.

Comment (m1pl) [reply .github/workflows/ci.yml:35]: @co-sic until then I'd probably take this out of the PR.

**Review by PeteMac88 — CHANGES_REQUESTED** [review]
I think we also need to add a CODEOWNERs file (https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners)

**Review by EnkhAmgalanGanbaatar — COMMENTED** [review]
LGTM

**Review by pommelinho — APPROVED** [review]

**Pull request merged**